packages/cli/docs-resolver/src/ApiReferenceNodeConverterLatest.ts

```diff
@@ -221,9 +222,7 @@ export class ApiReferenceNodeConverterLatest {
             this.#topLevelSubpackages.set(subpackage.id, subpackageNode);
             return subpackageNode;
         } else {
-            this.taskContext.logger.warn(
-                `Subpackage ${pkg.package} not found in ${this.apiDefinitionId}, treating it as a section`
-            );
+            this.taskContext.logger.warn(`Cannot find component ${pkg.title}, treating it as a section`);
```

Comment:
Can we update the message to: `Failed to locate API section plantInfo. Did you mean [a], [b] or [c] ?`